llvm/lib/Target/DirectX/DirectXRegisterInfo.h

```diff
@@ -22,6 +22,14 @@ namespace llvm {
 struct DirectXRegisterInfo : public DirectXGenRegisterInfo {
   DirectXRegisterInfo() : DirectXGenRegisterInfo(0) {}
   ~DirectXRegisterInfo();
+
+  const MCPhysReg *getCalleeSavedRegs(const MachineFunction *MF) const override;
```

Comment:
Had to implement these overrides to be able to create an instance of `DirectXRegisterInfo` in `llvm/lib/Target/DirectX/DirectXInstrInfo.h`